api/proto/teleport/legacy/types/types.proto

```diff
@@ -3298,6 +3299,13 @@ message AccessRequestSpecV3 {
   // LongTermResourceGrouping contains information about how resources can be grouped
   // based on Access List promotions for long-term Access Requests.
   LongTermResourceGrouping LongTermGrouping = 25 [(gogoproto.jsontag) = "long_term_grouping,omitempty"];
+
+  // the set of resources to which access is being requested,
+  // paired with additional information such as ResourceConstraints.
+  repeated ResourceAccessID RequestedResourceAccessIDs = 26 [
```

Comment:
Could you explain in comment what is the difference between `repeated ResourceID RequestedResourceIDs = 14 [` ?

Reply:
Gotcha, can do.